cdc/sink/producer/kafka/config.go

```diff
@@ -267,11 +279,6 @@ func newSaramaConfig(ctx context.Context, c *Config) (*sarama.Config, error) {
 		config.Producer.Compression = sarama.CompressionNone
 	}
 
-	// Time out in one minute(120 * 500ms).
-	config.Admin.Retry.Max = 120
-	config.Admin.Retry.Backoff = 500 * time.Millisecond
-	config.Admin.Timeout = 1 * time.Minute
```

Comment:
Any side effect of removing `Admin.Timeout`?

Reply:
when the `cli` try to create the changefeed, it will take effect. 

If the kafka is unavailable, this will return very fast.

Reply:
set `Admin.Timeout` to `10s`, try to make CI more stable.